docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.4'
+
+services:
+  skoruba.identityserver4.admin:
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
```

Comment:
If we really want to have different environmnet in Docker then let's have `Docker` environment instead of reusing Development env which is for development :)

This would also require to change `appsettings.Development.json` to `appsettings.Docker.json`

Reply:
In the future, I will adjust the database connection string to docker-compose.yml, appsettings.Development.json is not needed.

Reply:
Yeah since @skoruba added `AddEnvironmentVariables` to configuration you can nicely set it up in environment in docker compose :) I agree that this is superior approach.